packages/@okta/vuepress-site/docs/reference/api/schemas/index.md

```diff
@@ -850,27 +834,27 @@ curl -v -X POST \
 }
 ```
 
-### Update App User Profile Schema Property
+### Update App User Profile Schema property
 
 
 <ApiOperation method="post" url="/api/v1/meta/schemas/apps/${instanceId}/default" />
 
-Updates one or more [custom app user profile properties](#app-user-profile-schema-property-object) in the schema, or the nullability of a base property. Changing a base property's nullability (i.e., the value of its `required` field) is allowed only if it is nullable in the default predefined schema for the app.
+Updates one or more [custom App User Profile properties](#app-user-profile-schema-property-object) in the schema, or the nullability of a base property. Changing a base property's nullability (for example, the value of its `required` field) is allowed only if it is nullable in the default predefined Schema for the App.
 
-##### Request Parameters
+##### Request parameters
 
 
 | Parameter   | Description                                          | Param Type | DataType                                                       | Required | Default |
 | :----------- | :---------------------------------------------------- | :---------- | :-------------------------------------------------------------- | :-------- | :------- |
 | instanceId  | The id of the target App Instance                    | Path       | String                                                         | TRUE     |        |
```

Comment:
"ID"

Reply:
So, this actually needed to be `id` - because of the way it is represented in the code.